core/src/test/groovy/io/micronaut/core/convert/DefaultConversionServiceSpec.groovy

```diff
@@ -42,6 +42,7 @@ class DefaultConversionServiceSpec extends Specification {
         10                      | Float       | 10.0f
         10                      | String      | "10"
         "1,2"                   | int[]       | [1, 2] as int[]
+        "str"                   | char[]      | ['s', 't', 'r'] as char[]
```

Comment:
My test is failing... can you explain what I am doing wrong?

Reply:
The implementation isn't correct. The implementation you had before moving the converter is what it should have been